cacheray/src/cacheray_options.c

```diff
@@ -31,13 +31,23 @@ static void cacheray_options_set(cacheray_options_t *options, const char *key,
       fprintf(stderr, "Cacheray: tracefile error %d for '%s'\n", r, val);
       abort();
     }
+  } else if (!strcmp(key, "enabled")) {
+    if (!strcmp(val, "true")) {
```

Comment:
Can we just go with `0` and `1` and use `atoi` to convert?

Reply:
sure!

Reply:
I'm waiting for this change before re-review. Did you forget to fix this? Just so we're not waiting for each other 🙂

Reply:
I had to leave before implementing anything, first day at work :)

Anyway, I'm a little sceptical to using `atoi` since it has no error-handling and will output 0 (false) if it cannot convert the value into an int.

Reply:
Oh, happy firstday! :)

Yeah, `atoi` is a little crappy that way, but it would be nice not to carry around code for strict parsing at this stage. You could also do:

    options->enabled = !!strcmp(val, "0");

to get the familiar "0 means false, everything else true" semantics. I think I'd prefer that.

Reply:
Thanks!

Also, that solution is better. Will push fix 🙂